vizro-core/docs/pages/user-guides/card-button.md

```diff
@@ -602,12 +602,12 @@ Assign the predefined CSS variable `--inverse-color` to the `filter` property of
 
 ```css
 img[src*="#my-image"] {
-  filter: var(--icon-color);
+  filter: var(--inverse_color);
 }
 ```
 
-!!! example "Styled icon"
-    ![styled icon](../../assets/user_guides/components/responsive_icon.gif)
+??? example "Responsive icon"
+    ![responsive icon](../../assets/user_guides/components/responsive_icon.gif)
```

Comment:
Just curious: What was previously broken? And how does it fix it? Because here we just rename it right and change the admonition?

Reply:
Initially I thought the link was broken, but then realised the file itself was invalid/broken so I re-recorded the gif.

Reply:
And do you know why the gif initially was broken? Was it too big?

Reply:
No, it was not too big. I really don't know why it was broken, but it wouldn't run locally either.  Maybe I didn't save it correctly from snagit.